ui/views.py

```diff
@@ -107,17 +107,19 @@ class VideoViewSet(viewsets.ModelViewSet):
     def destroy(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         video = self.get_object()
         if request.GET.get("s3"):
-            video.s3_object.delete()
+            for file in video.videofile_set.all():
+                file.s3_object.delete()
```

Comment:
In my transcode branch I removed this and made it a signal, should I do that in this PR?
```
@receiver(pre_delete, sender=VideoFile)
@receiver(pre_delete, sender=VideoThumbnail)
def delete_s3_files(sender, **kwargs):
    """
    Make sure S3 files are deleted along with associated video file/thumbnail object
    """
    kwargs['instance'].s3_object.delete()
```